<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDU升级固件打包工具V1.8--数字签名</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="sFileEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>原始文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sFileBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="suffixEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>clever</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>后缀内容</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>转化</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>版本号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sVersionEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>29</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>降级选择：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="changeVercomboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>27</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>可降级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>不可降级</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>芯片选择：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chipVercomboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>STM32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>华大</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
